suite-native/module-authorize-device/src/components/connect/PinOnKeypad.tsx

```diff
@@ -0,0 +1,19 @@
+import { Image, VStack } from '@suite-native/atoms';
+
+import { deviceImageMap } from '../../constants/deviceImageConstants';
+import { PinForm, PinFormVariant } from './PinForm';
+
+type PinOnKeypadProps = {
+    variant: PinFormVariant;
+    onSuccess?: () => void;
+};
+
+export const PinOnKeypad = ({ variant, onSuccess }: PinOnKeypadProps) => {
+    return (
+        <VStack spacing="sp16" alignItems="center" flex={1} marginTop="sp24">
+            <Image source={deviceImageMap.T1B1} width={161} height={194} />
+            {/* TODO: Would it make sense to inline PinForm? */}
```

Comment:
So that we can have a discussion about it. 😉

`PinForm` is used exactly once and the component's props have to be duplicated here. It's just the `VStack` and image added here. So was thinking what to do about it.

I've got some preference but would like to hear your thoughts.

Reply:
No strong preference from me, but if I was coding this, I would probably have it in place if used only once.

Reply:
So up to you, just please remove the TODO after (not) doing it :)